python/lib/core/dmod/core/meta_data.py

```diff
@@ -536,41 +608,14 @@ def factory_init_from_restriction_collections(cls, data_format: DataFormat, **kw
                           continuous_restrictions=None if len(continuous) == 0 else continuous,
                           discrete_restrictions=None if len(discrete) == 0 else discrete)
 
-    def __eq__(self, other):
-        return self.__class__ == other.__class__ and self.data_format == other.data_format \
-               and self.continuous_restrictions == other.continuous_restrictions \
-               and self.discrete_restrictions == other.discrete_restrictions \
-               and self._custom_data_fields == other._custom_data_fields
-
     def __hash__(self) -> int:
-        cu = [] if self._custom_data_fields is None else [tup for tup in sorted(self._custom_data_fields.items())]
+        cu = [] if self.custom_data_fields is None else [tup for tup in sorted(self._custom_data_fields.items())]
         return hash((self.data_format.name,
```

Comment:
Typically Im a big fan of shorted line widths, but I prefer this current version. @robertbartel, what's your vote?

Reply:
I do like the current implementation better, although I fixed a typo just now and pushed another commit (still was using `self._custom_data_fields` instead of `self.custom_data_fields` in the second usage on the line).